webviz_config/_build_webviz.py

```diff
@@ -133,3 +136,21 @@ def run_webviz(args: argparse.Namespace, build_directory: pathlib.Path) -> None:
                 f"{terminal_colors.END}"
             )
             raise excep
+
+
+def _store_komodo_release(build_directory: pathlib.Path) -> None:
+    """Internally in Equinor, https://github.com/equinor/komodo is one platform used
+    for software deployment. For portable builds we want, until cloud hosting is common for
+    everyone, mention for the user which specific Komodo release (~= pip list) that was used
+    to build the portable Webviz application.
+
+    This function stores the Komodo release used, if Komodo is used, in a file komodo_release.txt.
+    """
+
+    if "KOMODO_RELEASE" in os.environ and sys.executable.startswith("/prog/res/komodo"):
+        (build_directory / "komodo_release.txt").write_text(
+            "This portable Webviz application was built with Komodo release:\n"
+            f"{os.environ['KOMODO_RELEASE']}\n\n"
+            "Try sourcing this Komodo release if you "
```

Comment:
Suggest that we explicitly state the required command to source the komodo release (probably best to use the csh alternative)